tgui/packages/tgui/interfaces/Tank.js

```diff
@@ -0,0 +1,81 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, NumberInput, ProgressBar, Section, Tooltip } from '../components';
+import { Window } from '../layouts';
+
+export const Tank = (props, context) => {
+  const { act, data } = useBackend(context);
```

Comment:
Not a blocker, but I still maintain that making the variables in `data` local so you don't have to prefix them by `data.` when using them is better in my opinion.

Reply:
I'm not willing to remove data. from these TGUI PRs, because I'm trying to maintain as much consistency with TG's version as possible, and renaming all the vars is not gonna help with that.